src/services/index.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+
+// Load `*.js` under current directory as properties
+//  i.e., `User.js` will become `exports['User']` or `exports.User`
+require('fs').readdirSync(__dirname + '/').forEach(file => {
+  if (file.match(/\.js$/) !== null && file !== 'index.js') {
+    const name: string = file.replace('.js', '');
+    const path: string = `./${file}`;
+    const Service = require(path);
+    module.exports[name] = new Service();
```

Comment:
Also, I just realized what this whole function does XD
I have been giving bad advises.
Do the manual import, don't be lazy XD you don't have that many services/routes.
By doing it explicit you are avoiding future bugs with this code :+1: just do the imports and exports como la gente
```js
const service = require('./servicio');
module.exports = {
  service,
};
```